airspeed/operators.py

```diff
@@ -12,6 +12,10 @@
 LOG = logging.getLogger(__name__)
 
 
+# This can be used by a function to bypass the silent behavior of a FormalReference.
+# If a function returns this value instead of None, the origial text of the reference will be returned instead.
+REPLACE_FORMAL_TEXT = "__FORMAL_REFERENCE__REPLACE_TEXT__"
```

Comment:
I am not sure about that name, I am welcome to a suggestion to make it more obvious as to what it does. 🤔